src/dbus.cpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * Copyright (C) 2021 YADRO.
+ */
+
+#include "dbus.hpp"
+
+#include "errors.hpp"
+
+#include <sdbusplus/exception.hpp>
+
+namespace phosphor
+{
+namespace dbus
+{
+
+sdbusplus::bus::bus bus = sdbusplus::bus::new_default();
+
+constexpr auto ObjectMapperService = "xyz.openbmc_project.ObjectMapper";
+constexpr auto ObjectMapperPath = "/xyz/openbmc_project/object_mapper";
+constexpr auto ObjectMapperIface = "xyz.openbmc_project.ObjectMapper";
+
+Services getObject(const Path& path, const Interfaces& ifaces)
+{
+    auto method = bus.new_method_call(ObjectMapperService, ObjectMapperPath,
+                                      ObjectMapperIface, "GetObject");
+    method.append(path, ifaces);
+
+    Services services;
+    try
+    {
+        bus.call(method).read(services);
+    }
+    catch (const sdbusplus::exception::SdBusError& ex)
+    {
+        if (0 == strcmp(ex.name(), "org.freedesktop.DBus.Error.FileNotFound"))
```

Comment:
I'd prefer `!strcmp()` as it's shorter and is still a well known idiom.

Reply:
It's a debatable assertion.
`!strcmp` looks like 'if comparison fails'. :)
But ok, I fixed it as you said.

Reply:
True, it's a moot point. I just prefer it that way for shortness. Thanks.